docs/attributes-registry/llm.md

```diff
@@ -0,0 +1,59 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: LLM
+--->
+
+# Large Language Model
+
+<!-- toc -->
+
+- [Generic LLM Attributes](#generic-llm-attributes)
+  - [Request Attributes](#request-attributes)
+  - [Response Attributes](#response-attributes)
+  - [Event Attributes](#event-attributes)
+
+<!-- tocstop -->
+
+## Generic LLM Attributes
+
+### Request Attributes
+
+<!-- semconv registry.gen_ai(omit_requirement_level,tag=llm-generic-request) -->
```

Comment:
Thanks @trisch-me, not sure I completely follow - should I create a separate namespace for `gen_ai.request`, `gen_ai.response`, etc.?

Reply:
since there is no final decision, I believe we can proceed with this PR. 
If a new process takes place before the PR is merged, we will be able to update it then.

Reply:
@nirga Yes, the idea of the issue is to have separate sub-namespaces, in your case `gen_ai.request` etc

But based on the data I see, that we will need to use tags anyway because one tag has different sub-namespaces, i.e. `gen_ai.request.*` is combined with `gen_ai.system`
cc @joaopgrassi

Reply:
Yeah here it is using tags to filter which table, instead of creating different groups. I'm fine to be like this, we can refactor it once we have decided.